components/Article/Page.js

```diff
@@ -180,6 +187,9 @@ class ArticlePage extends Component {
       secondaryNavExpanded: false,
       showSecondary: false,
       showAudioPlayer: false,
+      gallery: {
+        show: false
```

Comment:
Maybe it's just me, but I like to use the initial state to document potential state changes, so I would already include the startItem here:

```
gallery: {
  show: false,
  startItem: null
}
```